pom.xml

```diff
@@ -102,6 +105,26 @@
                 <artifactId>javax.annotation-api</artifactId>
                 <version>1.3.2</version>
             </dependency>
+            <dependency>
```

Comment:
Just double checking, we're not adding these as dependencies of the SDK, right?

Reply:
yes, they are part of `dependencyManagement`, so they will be included in child modules only if they are explicitly defined in `dependency` section of child modules.